tests/admin/permissions_test.py

```diff
@@ -0,0 +1,38 @@
+from pcapi.admin import permissions
+from pcapi.core.testing import override_settings
+import pcapi.core.users.factories as users_factories
+
+
+class HasPermissionTest:
+    def _make_user(self, email, is_admin=True):
+        return users_factories.UserFactory.build(email=email, isAdmin=is_admin)
+
+    @override_settings(PERMISSIONS="")
+    def test_empty_permission_list(self):
+        user = self._make_user("jane@example.com")
+        assert not permissions.has_permission(user, "unknown-permission")
+
+    @override_settings(
+        PERMISSIONS="\n".join(
+            ("can-frobulate: jane@example.com", "can-durlingate: john@example.com", "can-tergonize: roger@example.com")
```

Comment:
```suggestion
            ("can-frobulate: jane@example.com", "can-durlingate: john@example.com", "can-tergonize: roger@example.com, rabbit@example.com")
```
Profitons-en pour tester plusieurs emails.

Reply:
bonne idée, je prends ! :)